.travis.yml

```diff
@@ -6,7 +6,7 @@ sudo: required
 services: docker
 
 go:
-  - 1.13.3
+  - 1.13.8
```

Comment:
Closing this as the version is now updated to 1.14.2  https://github.com/etcd-io/etcd/pull/11864 Thanks!